inc/com/centreon/engine/host.hh

```diff
@@ -153,6 +160,29 @@ class                 host : public notifier {
   int                is_escalated_contact(contact* cntct);
   bool               is_result_fresh(time_t current_time, int log_this);
 
+  int                run_sync_host_check_3x(enum host::host_state* check_result_code,
```

Comment:
I think there is also run_sync_service_check(...) with the same api.
So it would be interresting to rename it run_sync_check(...). And then maybe we could push this new method to notifier.